src/assets/main.css

```diff
@@ -50,6 +50,10 @@ body {
   @apply h-full bg-gray-50 text-sm text-gray-700 dark:bg-gray-900 dark:text-gray-100;
 }
 
+#app {
+  @apply h-full;
+}
```

Comment:
It's possible just for the wizard to have `h-dvh`? This change might be a bit too much invasive